CMakeLists.txt

```diff
@@ -26,7 +26,12 @@ include(AbseilHelpers)
 
 
 # config options
-set(ABSL_STD_CXX_FLAG "-std=c++11" CACHE STRING "c++ std flag (default: c++11)")
+if (MSVC)
```

Comment:
I'm assuming that one of these options sets the default c++ standard to C++11? Abseil doesn't support earlier versions of the standard.

Reply:
VS 2015 and above only has C++14 (default) and C++17 (for VS 2017). They do not have C++11 mode.

See https://msdn.microsoft.com/en-us/library/mt490614.aspx